criu/include/restorer.h

```diff
@@ -177,6 +177,9 @@ struct task_restore_args {
 	pid_t				*zombies;
 	unsigned int			zombies_n;
 
+	int				*inotify_fds;
```

Comment:
we need a comment here which explains why we need inotify_fds in restore...

Reply:
done.